### PR TITLE
Update css

### PR DESCRIPTION
**Pull request by miamollie**

### CODE REVIEW

File: src/Field/index.js [hunk 1]
```diff
@@ -64,4 +65,9 @@ class Field extends React.Component {
   }
 }
 
+const fieldClasses = (): string => {
+  return classnames( mb2 )
```
Comment (margalit) [inline src/Field/index.js]: No space between brackets

File: src/Text/index.js [hunk 2]
```diff
@@ -1,50 +1,63 @@
...
+      [regular]: !props.bold,
+      [bold]: props.bold,
+      [italic]: props.italicised,
+      [textBlueDark]: !props.colour,
+      [textBlue]: props.colour === 'blue',
+      [textRed]: props.colour === 'red',
+      [textWhite]: props.colour === 'white',
+      [textGrey]: props.colour === 'grey',
+      [textGreyDark]: props.colour === 'greyDark',
+      [textGreyDarkOnSiblingInputFocus]: props.greyDarkOnSiblingInputFocus,
```
Comment (margalit) [inline src/Text/index.js]: Maybe this can be a better named prop?

File: src/Field/index.js (hunk 1 above)
Comment (miamollie) [inline src/Field/index.js]: Updated

File: src/Text/index.js (hunk 2 above)
Comment (miamollie) [inline src/Text/index.js]: Updated

File: docs/index.js [hunk 3]
```diff
@@ -47,12 +47,12 @@ const renderProps = (props: Object): Array<React.Element<any>> => {
     const prop = props[key]
     return (
       <Item key={index}>
-        <Text small blue bold>{key}</Text>
+        <Text size={'small'} colour={'blue'} bold>{key}</Text>
```
Comment (margalit) [inline docs/index.js]: This doesn't need to be surrounded by curly braces, _you're_ just passing a string so we can just pass it in as a string:

``` javascript
<Text size='small' colour='blue' bold>{key}</Text>
```

**Pull request merged**